pages/file-upload/permutations.page.tsx

```diff
@@ -0,0 +1,55 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import React from 'react';
+import createPermutations from '../utils/permutations';
+import PermutationsView from '../utils/permutations-view';
+import ScreenshotArea from '../utils/screenshot-area';
+import FileUpload, { FileUploadProps } from '~components/file-upload';
+import { i18nStrings } from './shared';
+
+const permutations = createPermutations<Omit<FileUploadProps, 'dismissAriaLabel' | 'i18nStrings'>>([
+  {
+    value: [[]],
+  },
+  {
+    value: [[new File([new Blob(['demo content'], { type: 'text/plain' })], 'demo file')]],
+  },
+  {
+    multiple: [true],
+    value: [
+      [
+        new File([new Blob(['demo content 1'], { type: 'text/plain' })], 'demo file 1'),
+        new File([new Blob(['demo content 2'], { type: 'text/plain' })], 'demo file 2'),
+      ],
+    ],
+  },
+  {
+    value: [[new File([new Blob(['demo content'], { type: 'text/plain' })], 'demo file')]],
+    showFileType: [true],
+    showFileSize: [true],
+    showFileLastModified: [true],
+    showFileThumbnail: [true],
```

Comment:
Can we include the `false` variants here as well? That will allow us to see all different combinations of these values on the page, and to catch any CSS issues in the screenshots.

Reply:
I am not sure if this is really needed those, adding true, false everywhere would produce lots of permutations. However, I can add [false, true] for the thumbnail because it renders differently compared to other atrributes.